CoreFoundation/RunLoop.subproj/CFRunLoop.c

```diff
@@ -4306,7 +4306,7 @@ CFIndex CFRunLoopTimerGetOrder(CFRunLoopTimerRef rlt) {
 
 void CFRunLoopTimerInvalidate(CFRunLoopTimerRef rlt) {	/* DOES CALLOUT */
     CHECK_FOR_FORK();
-    CF_OBJC_FUNCDISPATCHV(CFRunLoopTimerGetTypeID(), void, (NSTimer *)rlt, invalidate);
+    //CF_OBJC_FUNCDISPATCHV(CFRunLoopTimerGetTypeID(), void, (NSTimer *)rlt, invalidate);
```

Comment:
Please do not disable CF_OBJC_… functions. As the name implies, these are for ObjC (and are no-ops in Swift.)

Reply:
OK great, I didn't realise that. Specifically that one however does appear to be casting `CFRunLoopTimer` to `NSTimer *`, is that ok to keep in?